corva/models/scheduled.py

```diff
@@ -1,7 +1,12 @@
+from __future__ import annotations
+
 from datetime import datetime
-from typing import Optional
+from itertools import chain
+from typing import List, Optional
+
+from pydantic import Field, parse_raw_as
```

Comment:
It's better practice to import classes and modules instead of functions directly.

`import itertools`
`import pydantic`

Reply:
thanks, we would have to refactor whole project to fix all such issues